src/components/Button.jsx

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable react/prop-types */
+
+import '../styles/button.css';
+
+export default function Button(props) {
+    return (
+        <button className={props.classes} onClick={props.onButtonClick}>{props.buttonText}</button>
```

Comment:
Je zou i.p.v. props.buttonText ook children kunnen gebruiken. Die is wat veelzijdiger in het gebruik.